kong/plugins/rate-limiting/handler.lua

```diff
@@ -104,6 +111,18 @@ function RateLimitingHandler:access(conf)
         ngx.header[RATELIMIT_LIMIT .. "-" .. k] = v.limit
         ngx.header[RATELIMIT_REMAINING .. "-" .. k] = math.max(0, (stop == nil or stop == k) and v.remaining - 1 or v.remaining) -- -increment_value for this current request
       end
+
+      -- Find the closer limit.
+      local closer_limit = 0
+      for k, v in pairs(usage) do
+        if (closer_limit == 0 or closer_limit > v.remaining) then
+          closer_limit = v.remaining
+          ngx.header[RATELIMIT_LIMIT] = v.remaining
+          ngx.header[RATELIMIT_REMAINING] = v.limit
```

Comment:
Those headers duplicate what is already available (and more detailed) to the client. If the client wishes to compute what is the closest limit, they can do so themselves. Those new headers are unnecessary here.

Reply:
While "labeled" ratelimits are very useful, most of the clients will just honor `X-RateLimit-Limit|Remaining` with no suffixes. 

My goal is to provide the "vanilla" headers.

Delegating